symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -728,6 +729,18 @@ public V1AuditTrailInitiatorList listAuditTrail(@Nonnull Long startTimestamp, Lo
             startTimestamp, endTimestamp, before, after, limit, initiatorId, role));
   }
 
+  /**
+   * Suspends or re-activates a user account.
+   *
+   * @param userId  user id
+   * @param userSuspension action to perform on the user
+   * @see <a href="https://developers.symphony.com/restapi/reference#suspend-user-v1">Suspend User Account v1</a>
+   */
+  public void suspendUser(@Nonnull Long userId, @Nonnull UserSuspension userSuspension) {
```

Comment:
I was wondering if just "suspend" would be enough but we already have followUser/unfollowUser methods in this class

Reply:
At the end we only have one api to suspend, nothing similar. I guess we don't risk any misunderstanding.